### PR TITLE
Updated chart install instructions

### PR DESCRIPTION
**Pull request by vikram-bitnami**
**Description of the change**

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** [Place an '[X]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [ ] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

**Comment by carrodher** [comment]
You need to bump the chart version `Chart.yaml` according to [semver](http://semver.org/), in this case, patch version is enough.

File: bitnami/harbor/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ For example, the following changes have been introduced:
 ## TL;DR:
 
 ```
-helm repo add bitnami https://charts.bitnami.com
```
Comment (carrodher) [inline bitnami/harbor/README.md]: To be consistent with the rest of charts READMEs, can you add the `$` in the bash code blocks? It is a mistake since the beginning of the repository, I just found it and we can take advantage of this PR to fix it without bumping the chart version twice.

Comment (carrodher) [reply bitnami/harbor/README.md]: Don't worry about it, I've changed it

**Comment by vikram-bitnami** [comment]
> 
> 
> You need to bump the chart version `Chart.yaml` according to [semver](http://semver.org/), in this case, patch version is enough.

Done
